package.json

```diff
@@ -76,6 +76,7 @@
   "packageManager": "yarn@3.5.0",
   "dependencies": {
     "@linaria/babel-preset": "^4.4.3",
+    "@next/bundle-analyzer": "^14.2.5",
```

Comment:
Should this be added as a devDependency?

Reply:
It can be, but:
- doing so requires adding some prod-aware env var conditions to the next config, since the config is used for prod builds as well and we'd need to make sure this code isn't run when the dependency isn't available
- including it here doesn't affect the production bundle, since it's only used during build time
- in prod we don't run build with the `ANALYZE` flag, so the build artifacts aren't produced

I can switch it if you think it's better, it's a low-effort change and idc much either way, but for these reasons I chose to add as a normal dep initially.

Reply:
Cool, seems fine as is.